cpp/examples/README.md

```diff
@@ -0,0 +1,32 @@
+# libcugraph examples
+
+Example codes on how to use libcugraph.
+
+## Contents
+
+- single_gpu: simple code on how to use libcugraph to run different graph algorithms in a single GPU node.
+
+- multi_gpu: example code on how to use libcugraph to run different graph algorithms in a multi GPU node.
```

Comment:
We can run different algorithms using more than one node.

Reply:
changed.